tests/Domain/Purchase/Subscription/AutoRenewingPlanTest.php

```diff
@@ -48,4 +48,23 @@ public function instantiate(): void
         $this->assertInstanceOf(SubscriptionItemPriceChangeDetails::class, $actual->priceChangeDetails);
         $this->assertInstanceOf(InstallmentPlan::class, $actual->installmentDetails);
     }
+
+    /** @test */
+    public function instantiate_with_recurring_price_only(): void
+    {
+        $data = [
+            'recurringPrice' => [
+                'currencyCode' => $this->faker->currencyCode(),
+                'units' => (string)$this->faker->randomNumber(5),
+                'nanos' => $this->faker->randomNumber(5),
+            ],
+        ];
+
+        $actual = $this->normalizer->normalize($data, AutoRenewingPlan::class);
```

Comment:
[nitpick] Please add an empty line after the `Act` step to distinguish it from the following `Assert` section. [[AAA](https://github.com/openkairos/documentation/blob/main/docs/technical/unit-testing.md)]